src/main/java/com/kodilla/ecommercee/CartController.java

```diff
@@ -1,4 +1,44 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.CartDto;
+import com.kodilla.ecommercee.mapper.CartMapper;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/V1/cart")
 public class CartController {
+    private final CartMapper cartMapper;
+
+    public CartController(CartMapper cartMapper) {
+        this.cartMapper = cartMapper;
+    }
+
+    @GetMapping(value = "getProductsInCart")
+    public Map<Long, Integer> getProductsInCart(long cartId) throws CartNotFoundException {
```

Comment:
z ta metoda sa dwa problemy
- to powinno byc raczej wyszukiwanie karty po id. Najlatwiej bedzie zwracac CardDto
- unikaj zwracania mapy do frontu. W JS'ie trzymaja wszystko w tablicach i to jest dla nich (frontend'owcow) problem przy serializacji. Jesli juz musisz zwracac mape, to lepiej zamien ja na liste obiektow z kluczem i wartoscia